sources/packages/backend/libs/integrations/src/esdc-integration/student-loan-balances/student-loan-balances.processing.service.ts

```diff
@@ -79,9 +80,13 @@ export class StudentLoanBalancesProcessingService {
         );
     } catch (error) {
```

Comment:
Either the try catch needs to be global from line 76 - 126 or else it needs to return after logging at line 88. It cannot log and error and continue further.